sky/utils/cli_utils/status_utils.py

```diff
@@ -363,6 +364,13 @@ def _get_autostop(cluster_record: _ClusterRecord) -> str:
     return autostop_str
 
 
+def _get_head_ip(cluster_record: _ClusterRecord) -> str:
+    handle = cluster_record['handle']
+    if not isinstance(handle, backends.CloudVmRayResourceHandle):
+        return '-'
+    return handle.head_ip
```

Comment:
When the cluster is stopped, it shows `None` in the column. I guess we should show `'-'` instead?

Reply:
fixed